### PR TITLE
allow users to specify whether au-card is open on init or not

### PR DESCRIPTION
**Pull request by nvdk**
there's room for improvement here, but it works for the basic case.

### CODE REVIEW

**Comment by Windvis** [comment]
The only remark I have is that using the same argument name as the internal proprty name might be confusing when reading the code again in a few weeks :smile:.

**Comment by nvdk** [comment]
I've renamed the argument to isOpenInitially and added it to do the doc

**Comment by nvdk** [comment]
> Not sure if I understand what it does exactly or what the use case is but doesn't seem to break anything so ok for me. Do we need to add some extra information to the card component documentation page?

the idea is that in some uses cases the card can already be open without the user having to click, so it's still closable by the user if needs be. It's a somewhat limited use case, I mostly need it in GN atm for a not so pretty hack :see_no_evil:

File: tests/dummy/app/templates/docs/components/au-card.md [hunk 1]
```diff
@@ -149,7 +149,7 @@
 
 {{#docs-demo as |demo|}}
   {{#demo.example name='au-card-expandable.hbs'}}
-    <AuCard @expandable={{true}} @flex={{true}} @divided={{true}} as |c|>
+    <AuCard @expandable={{true}} @flex={{true}} @divided={{true} @isisOpenInitially={{true}} as |c|>
```
Comment (Windvis) [inline tests/dummy/app/templates/docs/components/au-card.md:152]: `isis` typo :grimacing:

File: tests/dummy/app/templates/docs/components/au-card.md [hunk 2]
```diff
@@ -149,7 +149,7 @@
 
 {{#docs-demo as |demo|}}
```
Comment (Windvis) [inline tests/dummy/app/templates/docs/components/au-card.md:150]: I think it should be added to the argument table at the bottom of the docs page as well?

**Review by Windvis — APPROVED** [review]

**Pull request merged**